src/pages/Scholarships.jsx

```diff
@@ -43,6 +45,14 @@ function ScholarshipsPage() {
         (loading && <CircularProgress className={classes.progress} />) || (
           <ScholarshipList scholarships={scholarships} />
         )}
+      <Box mt={24} mb={24} />
```

Comment:
This creates an awkward gap between the Load More button and the results of 192px, not 24px. Those [spacing properties](https://material-ui.com/system/spacing/#api) are multipliers on the theme's spacing. In this case, the theme's spacing is 8px as 8*24 = 192. You can double check this with the inspector.

Instead of a box I suggest adding top and bottom margins of 24px (3 * theme's spacing) to the Button.